### PR TITLE
new main.go

### PR DESCRIPTION
**Pull request by Xeonid**
new main.go

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -1,143 +1,46 @@
...
-		<-chanDone
-		fmt.Println("got bugs in", time.Since(tn))
-		time.Sleep(1 * time.Minute) // wait a bit for all http clients read data and put it into chanBug
-		close(chanBug)
-	}()
+func printHelp() {
+	fmt.Println("display available commands")
 }
 
-func readBugsFromFiles(ctx context.Context, chanBug chan bugz.Bug) error {
```
Comment (ngortheone) [inline main.go:124]: don't delete this function. leave it as is for now

File: main.go [hunk 2]
```diff
@@ -1,143 +1,46 @@
...
+	case "bugzilla-show-bugs":
+		showBugs()
+	default:
+		fmt.Println("use help")
 	}
-	err = db.GetPhabricatorData(ctx)
-	CheckFatal("getPhabricatorData", err)
 }
 
-func getBugsFromBugzilla(pullConcurrencyLevel int, chanBug chan bugz.Bug) {
```
Comment (ngortheone) [inline main.go:107]: don't delete this function. leave it as is for now

File: main.go [hunk 3]
```diff
@@ -1,143 +1,46 @@
...
-	"time"
-
-	"hugpoint.tech/freebsd/forge/bugz"
-	"hugpoint.tech/freebsd/forge/database"
-	"hugpoint.tech/freebsd/forge/git"
-	. "hugpoint.tech/freebsd/forge/util"
 )
 
 func main() {
-	fmt.Println("Awesome FreeBSD Phabricator to Gitea/Forgejo Migrator 9000")
```
Comment (ngortheone) [inline main.go:19]: save current contents of the main in some new function, say `oldmain`

**Review by ngortheone — CHANGES_REQUESTED** [review]

File: main.go (hunk 1 above)
Comment (Xeonid) [reply main.go:124]: done

File: main.go (hunk 2 above)
Comment (Xeonid) [reply main.go:107]: done

File: main.go (hunk 3 above)
Comment (Xeonid) [reply main.go:19]: made new file oldmain.go with old contents of the main

**Review by ngortheone — APPROVED** [review]

**Pull request merged**